_rules/SC1-2-video-audio-description.md

```diff
@@ -23,7 +18,7 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming][] video element [visible in the page][] where the video contains [audio][].
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio.
 
 ### Expectation
 
```

Comment:
"applicable target" instead of "test target" would be better in the expectation (because test targets might not be applicable)

Reply:
The test target is always the applicable elements. This is common language througout rules. I'd rather have that discussion separate from this PR. Please create a new issue if you think this is worth updating.